test/test_config.py

```diff
@@ -46,3 +51,30 @@ def test_overwrite_options():
         assert (
             config.ENVIRONMENT.MAX_EPISODE_STEPS == steps_limit
         ), "Overwriting of config options failed."
+
+
+CONFIGS_ALLOWED_TO_HAVE_NON_DEFAULT_KEYS = [
+    # new_keys_test.yaml excluded since it explicitely uses
+    # keys not present in the default for testing purposes
+    "configs/test/new_keys_test.yaml",
+    # Trainer excluded because does not use the default config
+    "configs/baselines/ppo.yaml",
+    # Planning Domain Definition Language configs are
+    # excluded since they do not implement the default config
+] + [glob("**/pddl/*.yaml", recursive=True)]
```

Comment:
glob already returns a list so this will be a list of list. This seems like a bug?
```suggestion
] + glob("**/pddl/*.yaml", recursive=True)